tests/tools.vitruv.applications.pcmumlclass.tests/src/tools/vitruv/applications/pcmumlclass/tests/PcmUmlclassViewFactory.xtend

```diff
@@ -0,0 +1,88 @@
+package tools.vitruv.applications.pcmumlclass.tests
+
+import tools.vitruv.testutils.TestViewFactory
+import tools.vitruv.framework.views.View
+import org.eclipse.uml2.uml.Model
+import org.eclipse.xtend.lib.annotations.FinalFieldsConstructor
+import org.palladiosimulator.pcm.repository.Repository
+import org.palladiosimulator.pcm.repository.RepositoryComponent
+import org.palladiosimulator.pcm.system.System
+
+@FinalFieldsConstructor
+class PcmUmlclassViewFactory extends TestViewFactory {
```

Comment:
Rename to `PcmUmlClassViewFactory`

Reply:
Renamed in 21ece0e47d03c405b02ba8337dcfc61c1a9fe2dd